### PR TITLE
test: jest migration 

### PR DESCRIPTION
**Pull request by juanpicado**
- /pnpm/packages/git-fetcher
- /pnpm/packages/global-bin-dir

### CODE REVIEW

File: packages/git-fetcher/package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   ],
   "scripts": {
     "_test": "cd ../.. && c8 --reporter lcov --reports-dir packages/git-fetcher/coverage ts-node packages/git-fetcher/test --type-check",
-    "test": "pnpm run compile && pnpm run _test",
+    "_test": "jest",
```
Comment (zkochan) [inline packages/git-fetcher/package.json:12]: seems not right

File: packages/git-fetcher/test/index.ts [hunk 2]
```diff
@@ -22,8 +20,8 @@ test('fetch', async t => {
       manifest,
     }
   )
-  t.ok(filesIndex['package.json'])
-  t.ok(await filesIndex['package.json'].writeResult)
-  t.equal((await manifest.promise).name, 'is-positive')
-  t.end()
+  expect(filesIndex['package.json']).toBeTruthy();
+  expect(filesIndex['package.json'].writeResult).toBeTruthy()
+  const name = (await manifest.promise).name;
```
Comment (zkochan) [inline packages/git-fetcher/test/index.ts]: we don't use semicolons

File: packages/global-bin-dir/package.json [hunk 3]
```diff
@@ -13,7 +13,7 @@
   },
   "scripts": {
     "lint": "eslint -c ../../eslint.json src/**/*.ts test/**/*.ts",
-    "test": "pnpm run compile && pnpm run _test",
+    "test": "jest",
     "prepublishOnly": "pnpm run compile",
     "compile": "rimraf lib tsconfig.tsbuildinfo && tsc --build",
     "_test": "cd ../.. && c8 --reporter lcov --reports-dir packages/global-bin-dir/coverage ts-node packages/global-bin-dir/test --type-check"
```
Comment (zkochan) [inline packages/global-bin-dir/package.json]: I think you had to update this script and keep test as it was

**Review by zkochan — CHANGES_REQUESTED** [review]

File: packages/git-fetcher/test/index.ts (hunk 2 above)
Comment (juanpicado) [reply packages/git-fetcher/test/index.ts]: Odd .. how this passed eslint ...

**Comment by juanpicado** [comment]
This is all for this PR, I'll continue later with other packages are a bit more complex.

**Pull request merged**